packages/koa-react-sidecar/src/graphql-client/createGraphQLClient.ts

```diff
@@ -0,0 +1,27 @@
+import {ApolloClient} from 'apollo-client';
+import {InMemoryCache, NormalizedCacheObject} from 'apollo-cache-inmemory';
+import {createApolloLink, Options} from './link';
+
+export type GraphQLClientOptions = Options & {
+  initialData?: NormalizedCacheObject;
+};
+
+export function createGraphQLClient({
```

Comment:
since this setup is only to connect to Shopify admin, and most of the rails + react actually connect to their own API, I dont think this setup will be useful for everyone.

I think we should instead let consumer pass their own `graphQLClient` or `createGraphQLClient`.